ansible/roles/stack-sunbird/templates/ml-survey-service.env

```diff
@@ -0,0 +1,154 @@
+#  AZURE_ACCOUNT_NAME=
+#  AZURE_ACCOUNT_KEY=
+#  AZURE_STORAGE_CONTAINER=
+#
+## Google Cloud Configuration
+#
+#  CLOUD_STORAGE=GCP
+#  GCP_PATH=
+#  GCP_BUCKET_NAME=
+#
+## AWS Cloud Configuration
+#
+#  CLOUD_STORAGE=AWS
+#  AWS_ACCESS_KEY_ID=
+#  AWS_SECRET_ACCESS_KEY=
+#  AWS_BUCKET_NAME=
+#  AWS_BUCKET_ENDPOINT=
+#  AWS_BUCKET_REGION=
+# To change the cloud config for azure, override below variable in common.yaml
+# For Example, 
+# ml_survey_cloud_config: |
+#   CLOUD_STORAGE=AZURE
+#   AZURE_ACCOUNT_NAME=mystorageaccount
+#   AZURE_ACCOUNT_KEY=xx123supersecretpassword
+#   AZURE_STORAGE_CONTAINER=slassessmentscontainer
+
+{{ml_survey_cloud_config}}
+
+# Mongodb connection url
+MONGODB_URL={{ml_survey_mongodb_url}}
+
+# Base url of the sunbird enviornment
+USER_SERVICE_URL={{ml_survey_user_service_URL}}
+
+# Current name for mongodb database 
+DB={{ml_survey_db_name|default('ml'+env)}}
+
+# Secure token access Admin related API's
+# The value of internal access token can be generated by command - "openssl rand -hex 10"
+INTERNAL_ACCESS_TOKEN={{ml_survey_internal_access_token}}
+
+# CSV Config
+# Create the respective directory as provided in the path below (Path to folder where reports will be stored)
+CSV_REPORTS_PATH={{ml_survey_csv_reports_path | default("reports")}}
+
+# Kafka Configuration
+# Flag which define where communication with kafka is on or off 
+KAFKA_COMMUNICATIONS_ON_OFF={{ml_survey_kafka_communications_on_off | default("ON")}}
+
+# IP address of kafka server with port without HTTP
+KAFKA_URL={{ml_survey_kafka_url}} 
+
+# Kafka topic name for pushing completed sumbmission
+COMPLETED_SUBMISSION_TOPIC={{ml_survey_completed_submission_topic}}
+
+# Kafka topic name for pushing incompleted submission
+INCOMPLETE_SUBMISSION_TOPIC={{ml_survey_incomplete_submission_topic}}
+
+# Kafka topic name for pushing submissions for which rating has to be done
+SUBMISSION_RATING_QUEUE_TOPIC={{ml_survey_submission_rating_queue_topic}}
+
+# Kafka topic name for pushing completed observation submission
+COMPLETED_OBSERVATION_SUBMISSION_TOPIC={{ml_survey_completed_observation_submission_topic}}
+
+# Kafka topic name for pushing incomplete observation submission
+INCOMPLETE_OBSERVATION_SUBMISSION_TOPIC={{ml_survey_incomplete_observation_submission_topic}}
+
+# Kafka topic name for pushing notification messages
+NOTIFICATIONS_TOPIC={{ml_survey_notifications_topic}}	
+
+# Kafka topic name for completed survey submission
+COMPLETED_SURVEY_SUBMISSION_TOPIC={{ml_survey_completed_survey_submission_topic}}
+
+# Kafka topic name for incomplete survey submission
+INCOMPLETE_SURVEY_SUBMISSION_TOPIC={{ml_survey_incomplete_survey_submission_topic}}
+
+# Kafka consumer group for ML Survey Service
+KAFKA_GROUP_ID={{ml_survey_kafka_group_id}}
+
+#  Kafka topic name for pushing project submission related data
+IMPROVEMENT_PROJECT_SUBMISSION_TOPIC={{ml_survey_improvement_project_submission_topic | default(env+".sl.improvement.project.submission")}}
```

Comment:
don't we have to rename this variable as well ?

Reply:
as per discussion with developer we are not going to rename it